README.md

```diff
@@ -9,25 +9,13 @@ The pyrevolve library is a thin Python wrapper around the reference implementati
 # Installation and usage
 To install, clone the repo and call
 
-    python setup.py build_ext --inplace
+    python setup.py build_ext
```

Comment:
Just a small note: I'd be good if we could clearly state that this will provide more than just a plain wrapper around the core C package.